.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @release-engineering
```

Comment:
This syntax is invalid.  The correct syntax/content was provided in the error message in your first PR commit at https://build.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Flibfabric/detail/PR-53/1/pipeline

```suggestion
# Have Release Engineering added as a reviewer to any packaging PR
* @daos-stack/build-and-release-watchers
```